src/sources/util/http.rs

```diff
@@ -124,6 +125,38 @@ impl HttpSourceAuth {
     }
 }
 
+fn decode(header: &Option<String>, mut body: Bytes) -> Result<Bytes, ErrorMessage> {
+    if let Some(encodings) = header {
+        for encoding in encodings.rsplit(',').map(str::trim) {
+            body = match encoding {
+                "identity" => body,
+                "gzip" => decode_read(GzDecoder::new(body.reader()), "gzip")?,
+                "deflate" => decode_read(DeflateDecoder::new(body.reader()), "deflate")?,
+                encoding => {
+                    return Err(ErrorMessage::new(
+                        StatusCode::UNSUPPORTED_MEDIA_TYPE,
+                        format!("Unsupported encoding {:?}", encoding),
```

Comment:
Why debug (`{:?}`) instead of display (`{}`)?

Reply:
Didn't thought it had `impl Display`.